prow/plugins/bugzilla/bugzilla.go

```diff
@@ -511,13 +517,29 @@ Comment <code>/bugzilla refresh</code> to re-evaluate validity if changes to the
 		log.WithError(err).Warn("Could not list labels on PR")
 	}
 	var hasValidLabel, hasInvalidLabel bool
+	var severityLabelToRemove string
 	for _, l := range currentLabels {
 		if l.Name == labels.ValidBug {
 			hasValidLabel = true
 		}
 		if l.Name == labels.InvalidBug {
 			hasInvalidLabel = true
 		}
+		if l.Name == labels.BZSevHigh || l.Name == labels.BZSevUrgent || l.Name == labels.BZSevMed || l.Name == labels.BZSevLow || l.Name == labels.BZSevUnspec {
+			severityLabelToRemove = l.Name
+		}
+	}
+	//handle severity labels
+	//Remove label that should not be set if set:
+	if severityLabelToRemove != "" && severityLabel != severityLabelToRemove {
+		if err := gc.RemoveLabel(e.org, e.repo, e.number, severityLabelToRemove); err != nil {
+			log.WithError(err).Error("Failed to remove severity bug label.")
+		}
+	}
+	if severityLabel != "" && severityLabel != severityLabelToRemove {
+		if err := gc.AddLabel(e.org, e.repo, e.number, severityLabel); err != nil {
+			log.WithError(err).Error(fmt.Sprintf("Failed to add severity bug label. :%v, %v", severityLabel, severityLabelToRemove))
```

Comment:
Use `.Errorf(` instead of `.Error(fmt.Sprintf`

We also don't need the labels in the error message, the `WithError()` will expose what we were trying to do with GitHub API anyway